api_server.py

```
@@ -503,10 +553,129 @@ async def chat(
+
+    Strategy B + Indic target language: buffers all chunks, translates the full
+    English answer, then emits a single translated chunk before the done event.
+    """
+    q: asyncio.Queue = asyncio.Queue()
+    loop = asyncio.get_running_loop()  # fix: get_event_loop() deprecated in async contexts (Python 3.10+)
+
+    def _run():
+        try:
+            for chunk in rag.llm_generate_stream(prompt, max_tokens):
+                loop.call_soon_threadsafe(q.put_nowait, ("chunk", chunk))
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)